### PR TITLE
Add Level 1 5G DES system

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
The 5G models were buried in the "level 3" type of GMT configuration with limited interconnections to other components. 

#### What does this PR accomplish?
This creates the 5G system with waste heat and GHX from a system parameter file and set of loads (ideally exported from OpenStudio). The 5G model is the series model that is in the MBL. It was imported into the GMT and templatized to replace the list of building loads.

#### How should this be manually tested?
* generate the 5G system `poetry run pytest ./tests/GMT_Lib/test_gmt_lib_des.py`
* Load the resulting package into Dymola and simulate. Package will be in the `./tests/GMT_Lib/output/DES_5G` directory.
* Rename SystemParameters.data to SystemParameters.param_template. There were two places were the instance data were being stored and it caused issues.

Note that none of the parameters are exposed and the partial model contains many of the setpoints for the waste heat and GHX. Those variables should be exposed to allow easy "setting".

### CODE REVIEW

File: geojson_modelica_translator/geojson/schemas.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import json
 import os
 
-from jsonschema.validators import _LATEST_VERSION as LatestValidator
+from jsonschema.validators import Draft202012Validator as LatestValidator
```
Comment (vtnate) [inline geojson_modelica_translator/geojson/schemas.py:7]: Pinning versions? Like a grown-up? 😮 ❤️

File: tests/data_shared/system_params_des_5g.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          "temp_hw_return": 35,
+          "has_liquid_heating": true,
+          "has_electric_heating": false,
+          "has_liquid_cooling": true,
+          "has_electric_cooling": false,
+          "temp_chw_supply": 7,
+          "temp_chw_return": 12
+        }
+      },
+      "ets_model": "Indirect Heating and Cooling",
```
Comment (vtnate) [inline tests/data_shared/system_params_des_5g.json:24]: This will need to be updated in the future. Tanushree is working on proper params for a 5g system.

**Review by vtnate — COMMENTED** [review]
Some in-progress thoughts. In general I love this.

File: geojson_modelica_translator/modelica/modelica_runner.py [hunk 3]
```diff
@@ -59,23 +58,34 @@ def _verify_docker_run_capability(self, file_to_run):
...
+            Path: Return the run_path as a Path object
         """
         if not run_path:
             run_path = os.path.dirname(file_to_run)
-        run_path = Path(run_path)
+        new_run_path = Path(run_path)
 
         # Modelica can't handle spaces in project name or path
-        if (len(str(run_path).split()) > 1) or (len(str(file_to_run).split()) > 1):
+        if (len(str(new_run_path).split()) > 1) or (len(str(new_run_path).split()) > 1):
```
[the remark this answers is not in the document]
Comment (nllong) [reply geojson_modelica_translator/modelica/modelica_runner.py:83]: oh, yeah, this is a mistake. nice catch! Fixed.

File: geojson_modelica_translator/modelica/modelica_runner.py [hunk 4]
```diff
@@ -84,14 +94,21 @@ def _copy_over_docker_resources(self, run_path: Path) -> None:
         shutil.copyfile(self.spawn_docker_path, new_spawn_docker)
         os.chmod(new_spawn_docker, 0o775)
 
-    def _subprocess_call_to_docker(self, run_path: Path, file_to_run: Str, action: Str, compiler: Str = 'optimica') -> int:
+    def _subprocess_call_to_docker(self, run_path: Union[str, Path], file_to_run: Union[str, Path], action: str, compiler: str = 'optimica') -> int:
```
[the remark this answers is not in the document]
Comment (nllong) [reply geojson_modelica_translator/modelica/modelica_runner.py:97]: done

File: geojson_modelica_translator/geojson/schemas.py (hunk 1 above)
Comment (nllong) [reply geojson_modelica_translator/geojson/schemas.py:7]: was more done for MyPy... but sure. haha.

File: tests/data_shared/system_params_des_5g.json (hunk 2 above)
Comment (nllong) [reply tests/data_shared/system_params_des_5g.json:24]: kk

**Comment by nllong** [comment]
@vtnate  -- one last check. If good, then feel free to merge.

**Review by vtnate — APPROVED** [review]

**Pull request merged**